woocommerce/wc-admin
# Deleted and re-generated package lock to fix packages

This fixes an issue where our local dependencies where being replace with the `npmjs` url of the release instead of using `file:packages/<component>`.
It wasn't until I tried wiping the package-lock and re-installing that this was correctly set in the package-lock. I know deleting the package-lock is kinda frowned upon, as it will also have updated packages where we didn't set the exact version. Any places where we made use of `~` or `^`, luckily we do use exact versions for most packages.

One additional change is the `sleep 10` in the e2e workflow, my guess is that the Database might not be up and fully running quite yet, although the [wait-for-build](https://github.com/woocommerce/woocommerce/blob/trunk/tests/e2e/env/bin/wait-for-build.sh) script succeeds.
You could have a quick scroll through the package-lock (I know it's long), but if any particular version updates stick out, that could be of concern.

### Testing instructions

- All actions of this PR should run successfully
- Pull down this PR, and clear the old node_module folders -> `./node_modules/.bin/rimraf ./packages/*/node_modules` and `sudo rm -r ./node_modules`. Verify your cache as well `npm cache verify`. 
- Check if the package-lock does not contain any versions that start with `file:https://npmjs`, instead there should be versions listed as `file:packages/<package name>`
- Run npm install, the package-lock should not be updated, but everything should install correctly. 
- Run a build and do a quick smoke test
